pkg/testhelper/testhelper.go

```diff
@@ -33,3 +38,36 @@ func ResultToString(result int) (str string) {
 	}
 	return ""
 }
+
+func SkipIfEmptyAny(skip func(string, ...int), object ...interface{}) {
+	for _, o := range object {
+		s := reflect.ValueOf(o)
+		if s.Kind() != reflect.Slice && s.Kind() != reflect.Map {
+			panic("SkipIfEmpty was given a non slice/map type")
+		}
+
+		if s.Len() == 0 {
+			skip(fmt.Sprintf("Test failed because there are no %s to test, please check under test labels", reflect.TypeOf(o)))
```

Comment:
All the traces int these two Skip* functions use the`"Test failed ..."` pattern? Shouldn't it be `"Test skipped ..."?`

Reply:
fixed